File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int sum[]=new int[2];
        int n=nums.length;
        for(int i=0;i<n;i++)
        {
            for(int j=i+1;j<n;j++)
            {
                if(nums[j]==target-nums[i])
                {
                    sum[0]=i;
                    sum[1]=j;
                    
                }
            }
        }

        return sum;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        numMap={}
        n=len(nums)

        for i in range(n):
            numMap[nums[i]]=i
        
        for i in range(n):
            diff=target-nums[i]

            if diff in numMap and numMap[diff]!=i:
                return [i,numMap[diff]]
        
        return []